test-configs.yaml

```diff
@@ -1016,6 +1016,23 @@ device_types:
     params: &uefi_i386
       bios_url: 'http://storage.kernelci.org/images/uefi/edk2-stable202005/i386/OVMF.fd'
 
+  qemu_riscv64:
+    base_name: qemu
+    mach: qemu
+    arch: riscv
+    boot_method: qemu
+    context:
+      arch: riscv64
+      cpu: 'rv64'
+      guestfs_interface: 'virtio'
+      machine: 'virt'
+      no_kvm: True
+      memory: 1024
+      extra_options: ['-bios default']
+    filters:
+      - passlist: {defconfig: ['defconfig']}
+    flags: ['qemu-riscv']
```

Comment:
This flag should probably be `qemu-riscv64` since you use "riscv64" in the rest of the naming.

Reply:
What is this flag for? As far as I can tell, it's not used anywhere.

Reply:
I wrongly understood flags? I have removed it